cmd/operator/api/v1/kubearchiveconfig_webhook.go

```diff
@@ -86,6 +87,9 @@ func (kaccv *KubeArchiveConfigCustomValidator) ValidateDelete(_ context.Context,
 
 func (kaccv *KubeArchiveConfigCustomValidator) validateKAC(kac *KubeArchiveConfig) (admission.Warnings, error) {
 	errList := make([]error, 0)
+	if kac.Namespace == constants.KubeArchiveNamespace {
+		return nil, fmt.Errorf("cannot create KubeArchiveConfig in the '%s' namespace", kac.Namespace)
+	}
```

Comment:
Why can't we create a `KubeArchiveConfig` in the `kubearchive` namespace? Also you may want to restrict this to the installation namespace instead, not the `kubearchive` namespace, as users can install on any namespace.

Reply:
So with this change we won't be able to create KubeArchiveConfigs in the installation namespace. I guess it is as good as any other moment to introduce this